### PR TITLE
Fixed usage of Directory.Build.props, added some more information to packages

### LINKED ISSUE #1228: Add 'Project site' and 'Source repository' links to dotvvm packages
Add links to manifest files so there are links from nuget.org (or from package explorer in VS) to the repo and project website.

* https://docs.microsoft.com/en-us/nuget/reference/nuspec#projecturl
* https://docs.microsoft.com/en-us/nuget/reference/nuspec#repository

### PR DESCRIPTION
**Pull request by acizmarik**
This PR adds some metadata into generated packages. Resolves #1228.
More importantly, this is a cleanup in csprojs and the way we use Directory.Build.props. All assemblies should now have correct metadata (previously it was not applied to some projects)

### CODE REVIEW

File: src/Framework/Hosting.Owin/DotVVM.Framework.Hosting.Owin.csproj [hunk 1]
```diff
@@ -7,24 +7,7 @@
...
-    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
-    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
-    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
     <PackageId>DotVVM.Owin</PackageId>
-    <Authors>RIGANTI</Authors>
-    <Description>
-      This package contains OWIN middleware and hosting infrastructure for DotVVM apps.
-      DotVVM is an open source ASP.NET-based framework which allows to build modern web apps without writing any JavaScript code.
-    </Description>
```
Comment (exyi) [inline src/Framework/Hosting.Owin/DotVVM.Framework.Hosting.Owin.csproj:23]:
```
    <Description>
      This package contains OWIN middleware and hosting infrastructure for DotVVM apps.
      $(Description)
    </Description>
```

File: src/Framework/Testing/DotVVM.Framework.Testing.csproj [hunk 2]
```diff
@@ -3,24 +3,7 @@
...
-    <GenerateAssemblyProductAttribute>false</GenerateAssemblyProductAttribute>
-    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
-    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
-    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
     <PackageId>DotVVM.Testing</PackageId>
-    <Authors>RIGANTI</Authors>
-    <Description>
-      This package contains helpers for testing DotVVM controls and other components.
-      DotVVM is an open source ASP.NET-based framework which allows to build modern web apps without writing any JavaScript code.
-    </Description>
```
Comment (exyi) [inline src/Framework/Testing/DotVVM.Framework.Testing.csproj:19]:
```
    <Description>
      This package contains helpers for testing DotVVM controls and other components.
      $(Description)
    </Description>
```

File: src/Directory.Build.props [hunk 3]
```diff
@@ -1,13 +1,44 @@
 ﻿<Project>
-  <PropertyGroup>
-    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
-    <AutomaticallyUseReferenceAssemblyPackages>false</AutomaticallyUseReferenceAssemblyPackages>
+  <PropertyGroup Label="Basic info">
+    <Authors>RIGANTI</Authors>
+    <Description>DotVVM is an open source ASP.NET-based framework which allows to build modern web apps without writing any JavaScript code.</Description>
```
Comment (exyi) [inline src/Directory.Build.props:4]:
```suggestion
    <Description>DotVVM is an open source ASP.NET-based framework which allows to build interactive web apps easily by using mostly C# and HTML.</Description>
```

**Review by exyi — APPROVED** [review]

**Comment by acizmarik** [comment]
This PR is completed and can be merged. 

I am not merging yet - in case we need to quickly create another release. Note that this change will most likely require adjustments to `ci/scripts/publish.ps1` before we can create new releases.

**Pull request merged**